packages/website/docs/docs/ui-elements.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Changing UI Elements
+description: Along with the editor itself, BlockNote includes menus and toolbars which you can replace or remove.
+imageTitle: Changing UI Elements
+path: /docs/ui-elements
+---
+
+
+<script setup>
+import { useData } from 'vitepress';
+import { getTheme, getStyles } from "./demoUtils";
+
+const { isDark } = useData();
+</script>
+
+# Changing UI Elements
+
+Along with the editor itself, BlockNote includes a few additional UI elements in the forms of menus and toolbars:
+
+- [Formatting Toolbar](/docs/formatting-toolbar)
+- Hyperlink Toolbar
+- [Slash Menu](/docs/slash-menu)
+- [Side Menu](/docs/side-menu)
+
+By default, these are all included in the editor, but you can remove or replace each of them with your own React components.
+
+## Positioner Components
```

Comment:
I would remove this part in favor of first showing the default contents of the <BlockNoteView>. Then you can use 1 sentence to explain what the positioners do, but it doesn't have to be a section